### PR TITLE
adding role-variable for relayhost

### PR DESCRIPTION
**Pull request by crazychatting**
this change will allow you to also easily change the value for relayhost.

### CODE REVIEW

**Comment by crazychatting** [comment]
Hello stale-bot,
please remove your stale label. This pull-request is still valid and I want @geerlingguy to review it.
Kind regards

File: defaults/main.yml [hunk 1]
```diff
@@ -6,3 +6,4 @@ postfix_service_enabled: true
 
 postfix_inet_interfaces: localhost
 postfix_inet_protocols: all
+postfix_relayhost:
```
Comment (geerlingguy) [inline defaults/main.yml]: This should default to an empty `''` (string), to be explicit.

File: README.md [hunk 2]
```diff
@@ -23,8 +23,9 @@ The state in which the Postfix service should be after this role runs, and wheth
 
     postfix_inet_interfaces: localhost
     postfix_inet_protocols: all
+    postfix_relayhost: mailgw.example.com
```
Comment (geerlingguy) [inline README.md]: The default listed in the README should match exactly the default that's in the `defaults/main.yml` file.

File: tasks/main.yml [hunk 3]
```diff
@@ -15,6 +15,8 @@
       value: "{{ postfix_inet_interfaces }}"
     - name: inet_protocols
       value: "{{ postfix_inet_protocols }}"
+    - name: relayhost
+      value: "{{ postfix_relayhost }}"
```
Comment (geerlingguy) [inline tasks/main.yml:19]: Is the setting `relayhost` okay to set without a value in the configuration? (e.g. as the default here), and is that the default for postfix out of the box? Just want to make sure if I push out a role update it doesn't break anyone's existing configuration.

**Review by geerlingguy — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 3 above)
Comment (crazychatting) [reply tasks/main.yml:19]: fair enough. I will check this. Thanks for the review!

File: defaults/main.yml (hunk 1 above)
Comment (crazychatting) [reply defaults/main.yml]: done

File: README.md (hunk 2 above)
Comment (crazychatting) [reply README.md]: done

File: tasks/main.yml (hunk 3 above)
Comment (crazychatting) [reply tasks/main.yml:19]: I tested this on a fresh install of postfix. This option is in the default main.cf uncommented and empty.

**Comment by crazychatting** [comment]
PR is still active

**Comment by crazychatting** [comment]
PR is still active

**Comment by crazychatting** [comment]
Hello stale-bot,
how can I reopen this PR? The PR is still valid.
Kind regards

**Pull request closed without merging**